src/Couchbase.Lite.Shared/API/Sync/ReplicatorConfiguration.cs

```diff
@@ -52,6 +52,28 @@ public enum ReplicatorType
         PushAndPull = Push | Pull
     }
 
+    /// <summary>
+    /// An enum representing level of opt in on progress of replication
+    /// </summary>
+    [Flags]
+    public enum ReplicatorOptionProgressLevel : int
```

Comment:
Can you point out where in the spec this is?  Was it removed recently?

Reply:
removed